source/controller/office.js

```diff
@@ -0,0 +1,67 @@
+import data from '../db/storage';
+
+class Office {
+  static async createOffice(req, res) {
```

Comment:
@Jesse-efe I'd like to ask these questions:

1. Why are you using async functions even when you are not performing any asynchronous task?